src/components/Toolbar.native.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+
+Test:
+
+  - Gestures disabled during toolbar scroll
+  - Overlay shown on hover/tap-and-hold after delay
+  - Overlay hidden on toolbar scroll
+  - Overlay hidden on touch "leave"
+
+*/
+
+import React, { useCallback } from 'react'
+import { shortcutById } from '../shortcuts'
+import { FlatList, ListRenderItem, TouchableOpacity, View } from 'react-native'
+// import { store } from '../store'
+import { TOOLBAR_DEFAULT_SHORTCUTS } from '../constants'
+import { Icon } from '../types'
+import HamburgerMenu from './HamburgerMenu'
+
+/** Toolbar component mobile. */
+const Toolbar = ({ dark = true, fontSize = 12, toolbarOverlay = null, scrollPrioritized = true, showTopControls = true }/* : ReturnType<typeof mapStateToProps> */) => {
+
+  const fillColor = dark ? 'white' : 'black'
+
+  const shortcutIds = TOOLBAR_DEFAULT_SHORTCUTS
+
+  const keyExtractor = useCallback((item: string) => item, [])
+  const renderItem: ListRenderItem<string> = useCallback(({ item: id, index }) => {
+
+    if (id === 'delete') return null
+
+    const { name, svg } = shortcutById(id)! ?? {}
+
+    const SVG = svg as React.FC<Icon>
+
+    return <TouchableOpacity style={{ margin: 8 }}>
+      <SVG key={name} fill={fillColor} size={35} />
+    </TouchableOpacity>
+
+  }, [])
+
+  /**********************************************************************
+   * Render
+   **********************************************************************/
+  return (
+    <View style={{ flexDirection: 'row', alignContent: 'center' }}>
```

Comment:
do we have a plan for styling?

Reply:
We could add our inline style linting rule, but there are a quite good number of components that have inline style. A possible solution would be to create a rule that only checks on files that finish with `*.native.ts`.